main/src/ca/uwaterloo/flix/api/lsp/provider/FindReferencesProvider.scala

```diff
@@ -167,7 +168,7 @@ object FindReferencesProvider {
     case SymUse.AssocTypeSymUse(_, loc) => loc.isReal
     case SymUse.CaseSymUse(_, loc) => loc.isReal
     case SymUse.DefSymUse(_, loc) => loc.isReal
-    case SymUse.EffectSymUse(_, loc) => loc.isReal
+    case SymUse.EffectSymUse(_, QName(_, _, loc)) => loc.isReal
```

Comment:
You use a mixture of styles, in some places its a deeper pattern match and in other places you write `qname.loc`. I think have a slight prefer for the latter, i.e. `qname.loc` instead of the deep match.